django/contrib/postgres/expressions.py

```diff
@@ -0,0 +1,12 @@
+from django.db.models.expressions import BaseExpression
+
+
+class Default(BaseExpression):
+    """
+    Expression for ``DEFAULT``.
+
+    In an insert query this will return the database default.
+    """
+
+    def as_sql(self, compiler, connection):
+        return connection.ops.pk_default_value(), []
```

Comment:
If we make this available for all backends, we should probably make some changes in this area:

- We should rename `pk_default_value()` to `default_value()`.
- We should have a `pk` argument to that so we can do `return 'NULL' if pk else 'DEFAULT'` for Oracle in that method.